src/site/_drafts/how-to-report-metrics/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
```

Comment:
Really interesting post.

My only real critique is that it's a lot of text. A few options to break that up might be to show screenshots from some of the services you mention. Or grab some public domain images that match your performance theme and sprinkle them throughout (check out [stock photos that don't suck](https://medium.com/@dustin/stock-photos-that-dont-suck-62ae4bcbe01b))

Reply:
I added one more. Woul love to show this chart as well:
https://discuss.httparchive.org/t/js-requests-size-1st-party-vs-3rd-party/1509
But not sure if I can, as it's from someone else (even though it's based on httparchive) . amybe with a source link?